hack/lib/init.sh

```diff
@@ -156,3 +156,28 @@ kube::realpath() {
   fi
   kube::readlinkdashf "$1"
 }
+
+function kube::build::ensure_docker_daemon_connectivity {
```

Comment:
yikes, you are correct, should have double checked the init.sh and what it sources out.